cleanlab/object_detection/rank.py

```diff
@@ -543,6 +578,7 @@ def _compute_overlooked_box_scores_for_image(
     lab_bboxes: Optional[np.ndarray] = None,
     similarity_matrix: Optional[np.ndarray] = None,
     min_possible_similarity: Optional[float] = None,
+    has_overlap_label_bboxes: Optional[np.ndarray] = None,
```

Comment:
[nit] `has_overlap_label_bboxes` sounds a bit more like a boolean flag instead of a vector. Super nit so feel free to ignore but maybe changing name of this variable to `overlap_label_bboxes` might be clearer. (I always confuse it with `overlapping_label_check` which sounds less like a boolean flag)

Reply:
Does badloc use "has_overlap_label_bboxes"? We should avoid passing this param into functions that don't use it (notice how high_probability_threshold is not passed into this function).

Reply:
the reason I passed it is: 
_get_valid_inputs_for_compute_scores_per_image is called in all of the functions, so 
has_overlap_label_bboxes is computed over and over again (3 times)